suite.go

```diff
@@ -43,8 +43,8 @@ const (
 	StepPending = models.Pending
 )
 
-type suite struct {
```

Comment:
Not necessarily against this change though I'm curious what prompted this particular change? IMO, naming something with a prefix of `test` _can_ denote a version of something used in tests. While it's true technically the suite is for testing is that something that one would imply? I suppose given the fact that you changed it, the answer to my last question is no? 😅

Reply:
It was simply to get the `suite` name out of the way so I could use it for the receiver. But if we're going back to `s` for the receiver then we don't need it.

Reply:
Resolved in 9297c4d6b28bada946c3e8ebf85be3ee87fb52c2